elastic/beats
# Update ES certificates for system tests

## What does this PR do?

Updates some TLS certificates on integration tests that expired on 6th Feb 2023

## Why is it important?

It fixes broken tests

## Checklist

~~- [ ] My code follows the style guidelines of this project~~
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

~~## Author's Checklist~~

## How to test this PR locally

Run the system tests, they should pass.
```
cd libbeat
mage -v PythonIntegTest
```

~~## Related issues~~
~~## Use cases~~
~~## Screenshots~~
~~## Logs~~
